.github/workflows/build-triton-wheel.yml

```diff
@@ -3,7 +3,7 @@ name: Build Triton wheels
 on:
   push:
     branches:
-      - main
+      - release/2.5
```

Comment:
Sorry, how is this a part of a revert?

Reply:
yes perhaps you right this is not required